README.md

```diff
@@ -56,7 +57,8 @@ All CLI functionality is available through `conduce/api.py` and all API function
 As an example, you may list existing datasets with the following command:
 
 ```
-conduce-api list datasets --host=app.conduce.com --user=email@example.com
+conduce-api list datasets --host=app.conduce.com
```

Comment:
Examples like this should be copy paste-able.  So we should either not include them in this change and allow them to scroll or escape the newlines with backslashes.  I think they are probably more readable on a single line.